parser/src/cfg/mod.rs

```diff
@@ -251,12 +271,60 @@ fn parse_cfg_raw(
     KeySeqsToFKeys,
     Overrides,
 )> {
-    let text = std::fs::read_to_string(p).map_err(|e| miette::miette!("{e}"))?;
-    let cfg_filename = p.to_string_lossy().to_string();
-    parse_cfg_raw_string(&text, s, &cfg_filename).map_err(error_with_source)
+    const INVALID_PATH_ERROR: &str = "The provided config file path is not a valid";
+
+    // IDEA: Maybe store the actual content of the files here
```

Comment:
I'm not against making this change if it's better, but currently not clear to me what the benefit of this would be.

Reply:
It was just a thought because I saw this as a possibility, but actually, I don't see any benefits from making this change too.